File: DotNet/Shared/Application/Models/PatientSubmissionModel.cs
using Hl7.Fhir.Model;

namespace LantanaGroup.Link.Shared.Application.Models
{
    public class PatientSubmissionModel
    {
        public string FacilityId { get; set; } = string.Empty;
        public string PatientId { get; set; } = string.Empty;
        public string ReportScheduleId { get; set; }
        public DateTime StartDate { get; set; }
        public DateTime EndDate { get; set; }
        public Bundle Bundle { get; set; }
    }
}

File: Java/shared/src/main/java/com/lantanagroup/link/shared/entities/PatientSubmissionModel.java
package com.lantanagroup.link.shared.entities;

import com.fasterxml.jackson.annotation.JsonFormat;
import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
import com.lantanagroup.link.shared.serdes.FhirIdDeserializer;
import lombok.Getter;
import lombok.Setter;
import org.hl7.fhir.r4.model.Bundle;

import java.util.Date;

@Getter
@Setter
public class PatientSubmissionModel {
    private String facilityId;

    @JsonDeserialize(using = FhirIdDeserializer.class)
    private String patientId;

    private String reportScheduleId;
    @JsonFormat(pattern = "yyyy-MM-dd'T'HH:mm:ss.SSSX", shape = JsonFormat.Shape.STRING)
    private Date startDate;
    @JsonFormat(pattern = "yyyy-MM-dd'T'HH:mm:ss.SSSX", shape = JsonFormat.Shape.STRING)
    private Date endDate;
    private Bundle bundle;
}
